pom.xml

```diff
@@ -365,6 +365,9 @@
                                 <xsdOption>
                                     <xsd>${basedir}/src/main/resources/xsd/oppdragskjema-v1.xsd</xsd>
                                 </xsdOption>
+                                <xsdOption>
+                                    <xsd>${basedir}/src/main/resources/xsd/grensesnittavstemmingskjema-v1.xsd</xsd>
+                                </xsdOption>
```

Comment:
Hvis denne xsd-en også skal brukes til konsistensavstemming, kan vi kanskje kalle fila for `avstemmingskjema-v1.xsd`? :)